lib/utils/validation.js

```diff
@@ -9074,6 +9081,7 @@ exports.maxCharacters = maxCharacters;
 exports.minAmount = minAmount;
 exports.minCharacters = minCharacters;
 exports.required = required;
+exports.requiredIf = requiredIf;
```

Comment:
not sure why are we doing this as old school, with exports in each function, and defining everything as a function() instead of `() => {}`, and using `var`...we shouldn't use it in that way, @davidpaley you created this file...why is it in this format without using ES6? It is not a bug @rin, but I would prefer consistency with the whole project

Reply:
This is because you are seeing in /lib, you should see in /src @kbardi . This is auto generated code

Reply:
hahaha, sorry, my bad 😭